### ISSUE TITLE
[Bug] Duplicate identifier 'Long'

### ISSUE THREAD

**Issue by marcoboucas** [temporalio/sdk-typescript#885]
### What are you really trying to do?

We were deploying a new version of our API and our CI worker failed with an error that seems to be linked to temporalio last version 1.3.0

### Describe the bug

It seems like the version 1.3.0 of @temporalio/protos has a duplicated const declaration on top of the file

Our package.json contains the following packages:
```
"@temporalio/client": "^1.1.0"
```

Complete version of the dependencies of our app
```
    "@google-cloud/speech": "^4.4.0",
    "@google-cloud/storage": "^6.1.0",
    "@google-cloud/text-to-speech": "^3.4.0",
    "@nestjs/axios": "^0.0.7",
    "@nestjs/common": "^8.4.2",
    "@nestjs/config": "^2.0.0",
    "@nestjs/core": "^8.4.2",
    "@nestjs/platform-express": "^8.4.2",
    "@nestjs/platform-socket.io": "^8.4.3",
    "@nestjs/schedule": "^1.0.2",
    "@nestjs/swagger": "^5.2.0",
    "@nestjs/terminus": "^8.0.6",
    "@nestjs/typeorm": "^8.0.3",
    "@nestjs/websockets": "^8.4.2",
    "@temporalio/client": "1.1.0",
    "@types/lodash": "^4.14.170",
    "@types/uuid": "^8.3.0",
    "class-transformer": "^0.4.0",
    "class-validator": "^0.13.2",
    "dotenv": "^8.2.0",
    "joi": "^17.4.0",
    "joi-to-swagger": "^6.0.1",
    "js-yaml": "^4.1.0",
    "lodash": "^4.17.21",
    "pg": "^8.5.1",
    "pumpify": "^2.0.1",
    "reflect-metadata": "^0.1.13",
    "rimraf": "^3.0.2",
    "rxjs": "^7.5.5",
    "socket.io": "^4.4.1",
    "swagger-ui-express": "^4.1.6",
    "tmp": "^0.2.1",
    "typeorm": "^0.2.31",
    "uuid": "^8.3.2"
```

We started having problems after the release of the 1.3.0 version of @temporalio/proto, and a switch back to the 1.1.0 version fixed the problem

![Capture d’écran 2022-09-21 à 11 09 13](https://user-images.githubusercontent.com/45031690/191925740-025d20da-4b46-458c-87bf-d987e8387f50.png)

### Minimal Reproduction

Build an app with nest, and in your package.json use the latest version of the temporalio/client seems to trigger an error for us

### Environment/Versions

- OS and processor: M1 Mac, Gitlab CI runner (node:16)
- Temporal Version: 1.3.0
- Are you using Docker or Kubernetes or building Temporal from source? no need to launch the temporal stack to trigger the error

### Additional context

We have fixed this problem temporary by overriding all the versions of @temporalio libraries to 1.1.0

**Comment by bergundy** [comment]
I see it, we'll issue a fix soon.
Thanks for reporting.

**Comment by bergundy** [comment]
Found the issue.
We were compensating for the fact that `pbts` didn't add the `long` import in our proto generation script and just recently this was fixed: https://github.com/protobufjs/protobuf.js/blame/d0268490cb08d2f88e0aafd1d0eb673bd2d3c714/cli/pbts.js#L173.

**Comment by marcoboucas** [comment]
This has been fixed in `1.4.0`, thanks a lot !
